src/hamiltonian_vib.py

```diff
@@ -304,6 +304,14 @@ def update_vibronic_hamiltonian(ham_el, ham_vib, params,E_SD,nac,suffix, opt):
                     ham_vib.set(I,J,(-1.0j+0.0)*nac.get(I,J))
                     print "\n"
       
+
+    if "inactive_states" in params:
```

Comment:
that doesn't look like the sufficient check - one could just define such a variable, but this won't mean param["inactive_states"] are initialized. Better, use something like
if len(params["inactive_states"]) > 0

Reply:
Okay, I'll use if len(params["inactive_states"]) > 0 . Thank you.